repo/packages/S/spark/12/marathon.json.mustache

```diff
@@ -39,6 +55,20 @@
     ],
     "container": {
         "type": "DOCKER",
+{{#security.authn.enabled}}
+        "volumes": [
+            {
+                "containerPath": "{{security.authn.ssl_files_dir}}",
+                "hostPath": "{{security.authn.ssl_files_dir}}",
```

Comment:
@mgummelt has this hardcoded to "/run/dcos/etc/agent/" in his branch, but that's clearly not right once we get these guys off the agent's cert. In fact, that directory should probably be root protected, and inaccessible to the spark scheduler once it can no longer run as root (tomorrow).
cc: @greggomann @alberts